build/main.cjs

```diff
@@ -2246,6 +2246,212 @@ async function importResponse(oldPtauFilename, contributionFilename, newPTauFile
 
 }
 
+/*
```

Comment:
How is this the content of this file different from the code added to cli.cjs?

Reply:
Both build/cli.cjs and build/main.cjs are generated by running `npm install && npm run build && npm run buildcli`. The main changes in source code are in cli.js, src/powersoftau.js and src/powersoftau_import_no_origin.js.